connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -198,6 +199,8 @@ private Long ensureLeaderConfig(long maxOffset) {
             }
         }
 
+        this.leaderState = new LeaderState(allConfigs, connectorAssignments, taskAssignments);
```

Comment:
Not directly related to this patch, but now that we have this recorded and easily usable, it would be nice to also expose this assignment state via the REST API.